experimental/auth-logic/BUILD

```diff
@@ -72,96 +60,122 @@ go_binary(
+        "//experimental/auth-logic/test_data:oak_provenance.json",
+    ],
+    outs = [
+        "oak_verification_passing.auth_logic",
+        "oak_verification_passing.auth_logic.dl",
+        "oak_verification_passing.csv",
+    ],
+    cmd = """
   export PROVENANCE=$(location //experimental/auth-logic/test_data:oak_provenance.json)
   export ENDORSEMENT=$(location //experimental/auth-logic/test_data:oak_endorsement.json)
   ./$(location :auth_logic_verification) oak_functions_loader:0f2189703c57845e09d8ab89164a4041c0af0a62 $$ENDORSEMENT $$PROVENANCE $(RULEDIR)/oak_verification_passing.auth_logic oak_verification_passing
   export SOUFFLE_BIN=$(location @souffle-bin//file)
   ./$(location @auth-logic-compiler//file) $(RULEDIR)/oak_verification_passing.auth_logic --souffle-output-file $(RULEDIR)/oak_verification_passing.auth_logic.dl --output_queries_directory $(RULEDIR)
   """,
-  outs = [
-    "oak_verification_passing.auth_logic",
-    "oak_verification_passing.auth_logic.dl",
-    "oak_verification_passing.csv"
-  ]
+    tools = [
+        ":auth_logic_verification",
+        "@auth-logic-compiler//file",
+        "@souffle-bin//file",
+    ],
 )
 
 go_test(
     name = "oak_auth_logic_verification_test",
     size = "medium",
     srcs = ["oak_auth_logic_verification_test.go"],
-    embed = [":auth_logic_interface"],
     data = [
-      "verification_success.csv",
+        "verification_success.csv",
     ],
+    embed = [":auth_logic_interface"],
 )
 
 go_test(
     name = "oak_verification_passing_test",
     size = "medium",
     srcs = ["oak_verification_passing_test.go"],
-    embed = [":auth_logic_interface"],
     data = [
-      "oak_verification_passing.csv",
+        "oak_verification_passing.csv",
     ],
+    embed = [":auth_logic_interface"],
 )
 
 # This rule runs the authorizaiton logic compiler on "simple.auth_logic"
-# as an input and produces the resulting souffle code and CSVs containing the 
+# as an input and produces the resulting souffle code and CSVs containing the
 # results of queries. These outputs are used by "simple_auth_logic_test"
 genrule(
-  name = "compile_simple_auth_logic",
-  srcs = ["simple.auth_logic"],
-  tools = [
-    "@auth-logic-compiler//file",
-    "@souffle-bin//file"
-  ],
-  cmd = """
+    name = "compile_simple_auth_logic",
+    testonly = True,
+    srcs = ["simple.auth_logic"],
+    outs = [
+        "simple.auth_logic.dl",
+        "demo_working.csv",
+        "demo_disappointing.csv",
+    ],
+    cmd = """
     export SOUFFLE_BIN=$(location @souffle-bin//file)
     ./$(location @auth-logic-compiler//file) $(location simple.auth_logic) --output_queries_directory $(RULEDIR) --souffle-output-file $(RULEDIR)/simple.auth_logic.dl
   """,
-  outs = ["simple.auth_logic.dl", "demo_working.csv", "demo_disappointing.csv"]
+    tools = [
+        "@auth-logic-compiler//file",
+        "@souffle-bin//file",
+    ],
+)
+
+go_library(
+    name = "lib",
```

Comment:
Is it common to name things `lib`? I have usually seen the main rule of a package being named with the same name as the package itself.

Reply:
This rule was autogenerated by gazelle. It's gone now.